src/consts/consts.py

```diff
@@ -294,6 +294,16 @@ class Platforms:
     VSPHERE = "vsphere"
 
 
+class PlatformTypes:
```

Comment:
```suggestion
class PlatformType:
```

I know some of those classes in this file are following either singular or plural, I think the former is more readable and ultimately we should move other classes to this form

Reply:
The problem is the the service expects the new format (i.e. VSphere), so platform.lower() won't work when passing it to the service. That's why we need the conversion method between the formats. Can we do it nicer?